src/main/java/com/google/cloud/genomics/utils/Contig.java

```diff
@@ -148,5 +184,78 @@ public Contig apply(String contigString) {
 
     return contigs;
   }
+  
+  /**
+   * Get a list of Contigs representing sharded windows of all genomic regions within the variant set.
+   * 
+   * These are useful for pipelines that operate in parallel against data within a specific variant set.
+   * 
+   * @param genomics - the genomics client
+   * @param variantSetId - the id of the variant set for which to create shards
+   * @param sexChromosomeFilter - whether or not to include the sex chromosomes in the list of shards
+   * @return a shuffled list of shards
+   * @throws IOException
+   */
+  public static List<Contig> getAllShardsInVariantSet(final Genomics genomics, final String variantSetId,
```

Comment:
I would not include this form of the function. It's not needed in a minimal API, and adds work for us. Same for all other variants that pick DEFAULT_NUMBER_OF_BASES_PER_SHARD instead of getting it via argument.